feat(dev): expose APIs for client-server communication (#7437)
vitejs/vite: docs/guide/api-plugin.md

normalizePath('foo\\bar') // 'foo/bar'
normalizePath('foo/bar') // 'foo/bar'
```

## Client-server Communication

> As in Vite 2.9, we provides some utilities for plugins to handle the communcation with clients easier.

Author: ```suggestion
As in Vite 2.9, we provide some utilities for plugins to handle the communication with clients easier.
```

> As in Vite 2.9, we provide some utilities for plugins to handle the communication with clients easier.

Reviewer: ```suggestion
Since Vite 2.9, we provide some utilities for plugins to help handle the communication with clients.
```
